### ISSUE TITLE
Naming consistency in lag/lacp models

### ISSUE THREAD

**Issue by ashutshkumr** [open-traffic-generator/models#249]
- In `lag_metrics`, request column name states `oper_states` but response column name states `oper_status`. Need to stick to one name
  - Why can't use `up_links` instead of `member_ports_up` ? Since it's more consistent with `min_links` (and both of these attributes decide up oper_status of lag, hence making them related).
- In `lacp_lag_member_metrics`,
  - Could we just call it `lacp_member_metrics` (since LAG is implied from the keyword `lacp`) ?
  - Could use `member_port_name` instead of `lag_member_port_name`
  - Use either `in`/`out` or `tx`/`rx/`, not both (e.g. `lacp_in_pkts` vs `lacp_rx_errors`)
    - IMHO, using `tx`/`rx` should be fine: `lacp_packets_tx`, `lacp_packets_rx`, `lacp_rx_errors`
- In `set_device_state`, `member_port_names` can be used instead of `lag_member_port_names`

**Comment by ashutshkumr** [comment]
@ajbalogh @PrasenjitAdhikary could you please help review wrt to naming consistency ? the idea is to not redundantly use `lag` and `lacp` together and make some column names more consistent with what we've already been doing in OTG.

**Comment by PrasenjitAdhikary** [comment]
@apratimmukherjee to help with reviewing the naming consistency.

**Comment by PrasenjitAdhikary** [comment]
We had a debate earlier, in the initial proposal document review and decided "lacp member" is incorrect terminology. Instead, the member is applicable for LAG only, and hence use of 'lag' and 'lag-member'

**Comment by PrasenjitAdhikary** [comment]
In stats name, `in`/`out` and `tx`/`rx` both are used for consistency with openconfig stats name. Following are the openconfig names:
```
                    +--ro counters
                       +--ro lacp-in-pkts?               oc-yang:counter64
                       +--ro lacp-out-pkts?              oc-yang:counter64
                       +--ro lacp-rx-errors?             oc-yang:counter64
                       +--ro lacp-tx-errors?             oc-yang:counter64
                       +--ro lacp-unknown-errors?        oc-yang:counter64
                       +--ro lacp-errors?                oc-yang:counter64
                       +--ro lacp-timeout-transitions?   oc-yang:counter64
```

**Comment by PrasenjitAdhikary** [comment]
Agreed on the following points, after a discussion: (Prasenjit, Kingshuk, Ashutosh, Subrata, Sourav) 
-	oper_status : Correct request column name to match with response name.
-	member_ports_up : No change.
-	lacp_lag_member_metrics : change to “lacp_metrics”
-	lag_member_port_name : No change
-	in/out/tx/rx : Use tx/rx  instead of in/out in OTG model. No change in telemetry name.

**Comment by ankur-sheth** [comment]
Need this resolved before v1.0.0.
